install/helm/agones/templates/service.yaml

```diff
@@ -28,4 +28,6 @@ spec:
     stable.agones.dev/role: controller
   ports:
     - port: 443
-      targetPort: 8081
\ No newline at end of file
+      targetPort: 8081
```

Comment:
This needs a name - otherwise:

```
Error: release agones failed: Service "agones-controller-service" is invalid: spec.ports[0].name: Required value
Makefile:195: recipe for target 'install' failed
make: *** [install] Error 1
```

:crying_cat_face:

Reply:
TIL: port needs name if there are multiple of them.

Reply:
fixed